docs/running-on-yarn.md

```diff
@@ -82,6 +82,19 @@ In `cluster` mode, the driver runs on a different machine than the client, so `S
 
 Running Spark on YARN requires a binary distribution of Spark which is built with YARN support.
 Binary distributions can be downloaded from the [downloads page](https://spark.apache.org/downloads.html) of the project website.
+There are two variants of Spark binary distributions you can download. One is pre-built with a certain
```

Comment:
no really biggie but I personally avoid abbreviation in the public documentation.

Reply:
In the download link, we use terminology of `pre-built` and `pre-built with user provided hadoop`, and in some other places, we use `with-hadoop`, and `no-hadoop` distributions. That was why I use both of them here to make it clear.

Reply:
Oops, I meant things like "don't"